modules/core/src/main/java/org/apache/fluo/core/oracle/OracleServer.java

```diff
@@ -161,11 +161,20 @@ public void run() {
           }
         }
       };
+      TimerTask logTask = new TimerTask() {
+        @Override
+        public void run() {
+          log.warn("Loging the current timestamp: ", currentTs);
+        }
+      };
+
       timer = new Timer("Oracle gc update timer", true);
       long updatePeriod =
           env.getConfiguration().getLong(FluoConfigurationImpl.ZK_UPDATE_PERIOD_PROP,
               FluoConfigurationImpl.ZK_UPDATE_PERIOD_MS_DEFAULT);
+      long nextPeriod = 5 * 60 * 1000L;
       timer.schedule(tt, updatePeriod, updatePeriod);
+      timer.schedule(logTask, nextPeriod, nextPeriod);
```

Comment:
Would be nice to set the delay to 0 instead of nextPeriod, so that when server starts up it logs the initial timestamp.